deps/v8/include/v8.h

```diff
@@ -8702,9 +8717,7 @@ class V8_EXPORT V8 {
    * Sets V8 flags from a string.
    */
   static void SetFlagsFromString(const char* str);
-  static void SetFlagsFromString(const char* str, size_t length);
-  V8_DEPRECATED("use size_t version",
-                static void SetFlagsFromString(const char* str, int length));
```

Comment:
@addaleax Can't we keep the deprecation? It shouldn't break users, I think?
This change breaks V8 compilation:
```
10:08:36 ../../src/api/api.cc:892:27: error: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'int' [-Werror,-Wshorten-64-to-32]
10:08:36   SetFlagsFromString(str, strlen(str));
10:08:36   ~~~~~~~~~~~~~~~~~~      ^~~~~~~~~~~
10:08:36 1 error generated.
```

Reply:
@targos The issue with that is that if both the `size_t` and the `int` variants are provided, a call to `SetFlagsFromString()` with an argument that is neither type, but convertible to both, (e.g. `unsigned int`) will give a compile error, so having both overloads is actually an API-breaking change. We should probably just add an explicit static_cast here (I can’t push right now, but will do as soon as I can if nobody else beats me too it).